### ISSUE TITLE
Unread messages outside the Inbox?

### ISSUE THREAD

**Issue by emoreau99** [psxpaul/office365-checker#1]
Hi

Would it be possible to show the count (and be warned) when new messages hit folders? That would be really nice. Otherwise, for me, it is useless.

I have rules that automatically move emails to folders when I receive them but your tool just ignore them.

**Comment by psxpaul** [comment]
Yes, I believe it should be possible. I would want it to be a configurable feature personally, since I use my email folders in the opposite manner - things I don't need to read right away are moved out of my inbox.

I think you would need to create an options page, which would fetch the list of folders in your inbox, and let you specify which ones to check. I'm not sure if the EWS API allows you to check the read message count of multiple folders at once, so you might need to make multiple calls every time you check for new messages.

This seems like a substantial amount of work, and I probably won't have time to work on this for at least a couple of weeks. Feel free to submit a pull request. I can answer any questions you have about the code base as well.
